wiki/Contests/Monthly_Beatmapping_Contest/en.md

```diff
@@ -0,0 +1,79 @@
+# Monthly Beatmapping Contest
+
+As implied by its name, the Monthly Beatmapping Contest (_MBC_) is a contest that runs monthly and allows users to compete with their beatmapping capabilities. 
+
+The MBC was originally managed on a nearly monthly basis between 2013 and 2015 and was followed by hiatus until 2020. The first 12 iterations of the MBC can be found on the [Contests](/wiki/Contests#monthly-beatmapping-contest) article. The results for the 13th iteration can be found [here](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results).
+
+Since 2020 the MBC has adopted a new schedule, ruleset, voting procedure, and reward scheme. The rest of this article details the current format for Monthly Beatmapping Contests.
+
+## Schedule
+
+The timeline for each month's contest is as follows:
+
+- **Beatmapping (3 weeks)**: Contest is announced at the start of the month and beatmappers create their entries.
+- **Screening (1 week)**: Entries are evaluated by experienced beatmappers.
+- **Voting (2 weeks)**: Top entries from screening are published for the osu! community to vote on.
+- **Rewards**: Winners are announced and prizes are redeemed.
+
+The voting portion of each contest overlaps with the next month's contest's beatmapping portion.
+
+## Ruleset
+
+Submissions for the MBC must abide by the following rules:
+
+- **Submissions must follow the month's beatmapping limitation.** The limitation differs month by month and is detailed on each contest's announcement post.
```

Comment:
Is this point about submitting multiple times using different songs? I was going to ask a similar question, but if so, this would answer it.

Reply:
it's a gimmick for the contest, like "your map must use circles only"
unrelated to songs or multiple submissions